docs/api/standard_library_types.md

```diff
@@ -51,7 +51,7 @@ except ValidationError as e:
 Pydantic supports the following [datetime](https://docs.python.org/library/datetime.html#available-types)
 types:
 
-### `datetime.datetime`
+### [`datetime.datetime`][]
```

Comment:
Not sure you'd want references directly on the headings. If so, I can remove them

Reply:
🤷‍♀️ I think it'd be nice! Looks good.

Reply:
Just checking - have you built the docs locally to confirm that the `[`datetime.datetime`][]` style of references correctly redirects to the stdlib python docs?

Reply:
Just a note: these manual references will produce warnings if they can't be resolved, failing MkDocs builds in strict mode.